ligo/stablecoin/fa1.2/actions.ligo

```diff
@@ -681,16 +681,19 @@ function add_permit
 function set_expiry
   ( const param: set_expiry_param
   ; const store : storage
+  ; const full_param: closed_parameter
   ) : entrypoint is block
-{ const new_expiry : seconds = param.0
+{ const owner : address = param.issuer
+; const updated_store : storage = sender_check(owner, store, full_param, "NOT_PERMIT_ISSUER")
+
 ; const updated_permits : permits =
-    case param.1 of [
+    case param.permit_hash of [
     | None ->
-        set_user_default_expiry(Tezos.get_sender(), new_expiry, store.permits)
-    | Some(hash_and_address) ->
-        set_permit_expiry(hash_and_address.1, hash_and_address.0, new_expiry, store.permits, store.default_expiry)
+        set_user_default_expiry(owner, param.expiry, updated_store.permits)
```

Comment:
It seems that the call to `set_user_default_expiry` is made with `param.issuer` while previously it was called with `Tezos.get_sender()`. Is this change intentional?

Reply:
>  Is this change intentional?

Yes, that was the same change we applied to `actions.ligo` for the FA2 variant of the contract in #134 (see the diff [here](https://github.com/tqtezos/stablecoin/pull/134/files#diff-3f97ef02a46298c9eb30b26bd76a27da8fea70f49db7d54799b3c814eda91357L761-R768)), but back then we forgot to apply the same changes to the FA1.2 variant. 

The description of the first commit in this PR contains some details about this as well.